src/ProjectTemplates/Microsoft.Extensions.AI.Templates/templates/McpServer-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -1,11 +1,6 @@
 {
   "$schema": "https://json.schemastore.org/dotnetcli.host",
   "symbolInfo": {
-    "TargetFrameworkOverride": {
-      "isHidden": "true",
-      "longName": "target-framework-override",
-      "shortName": ""
-    },
```

Comment:
This option was part of the first version of the template, with it coming from dotnet/templating sample code, but it's not in use, it was hidden, and it's undocumented, so it's safe to remove.